general/clients/css-customization.md

```diff
@@ -5,75 +5,211 @@ title: CSS Customization
 
-## Color Modifications
+If you have little or no experience with CSS, various resources and tutorials can be found online, together with using the below modifications as examples it is quite easy to get started making your own changes to your Jellyfin instance.
 
-### Colors Codes:
+![](~/images/custom-css-customcssfield.png)
 
-Here are a few examples, see [here](https://htmlcolorcodes.com/color-picker/) for a hex color chart.
+## General information about CSS
 
-Green: `#5dd000`<br>
-Blue: `#0000d0`<br>
-Red: `#d00000`<br>
+You can learn more about CSS using sites like [w3school](https://www.w3schools.com/css/default.asp). Below are some very basic details that will let you do rudimentary edits to the ready made modifications below. 
 
-#### Played Indicator:
+### Colors
+
+CSS supports multiple color formats, most typically hex is used. But simply text works too. To get "yellow" you can simply write "yellow", this will use preset yellow color. For more specificity, see below.
+
+Some examples of hex color codes.
+
+`#5dd000` Green <br> 
+`#0000d0` Blue <br>
+`#d00000` Red <br>
+`#00000058` Transparent black
+
+Go [here](https://htmlcolorcodes.com/color-picker/) for a hex color chart.
+
+### Comments
+
+A section of code or text inbetween `/*` and `*/` indicate a comment, and will be ignored. This allows you to add descriptions for what any particular section of code does to make it easily identifiable. It can also be used to disable code without deleting it. For example
+
+`/*This might be added above code to tell you what it does*/`
+
+### CSS Chaining
+
+CSS can be chained together to modify different sections together. An example of this is the "Border color" mod. It lists elements to be modified, and performs a change that is applied to all of them.
+
+## Modifications list
+
+To apply any one of these, copy paste the CSS code into the "Custom CSS" field. To use multiple modifications, simply add them one after another into the field. Any applied code will remain in the field. To remove a modification, delete or comment out the code for it from the field. Changes apply immediately when the settings page is saved and doesn't require restarting.
+
+#### Played Indicator
+
+This will affect the played/watched indicator. Replace the color hex with any value you like.
+
+**Without mod**
+
+![](~/images/custom-css-normalwatched.png)
+
+**Green**
 
 ```css
 .playedIndicator { background: #5dd000; }
 ```
 
-Before:
+![](~/images/custom-css-greenwatched.png)
+
+**Transparent dark using RGBA hex value**
+
+```css
+/*Make watched icon dark and transparent*/
+.playedIndicator {background: #00000058;}
+```
+
+![](~/images/custom-css-transparentwatched.png)
+
+#### Make top menu transparent
```

Comment:
```suggestion
#### Transparent Top Menu
```

Reply:
I usually prefer that headers always have title casing.

Reply:
Applied change to all headers